bundles/io/org.eclipse.smarthome.io.rest.core/OSGI-INF/configurableserviceresource.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource">
+   <implementation class="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource"/>
+   <service>
+      <provide interface="org.eclipse.smarthome.io.rest.RESTResource"/>
+      <provide interface="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource"/>

```

Comment:
I does not complain, that it register an OSGi service, but my experience has been, that an OSGi service is provided by using an interface. But I have read the part of the compendium spec and it could also be a class.

```text
The name of the interface that this service is registered under. This
name must be the fully qualified name of a Java class. For example,
org.osgi.service.log.LogService. The specified Java class should be an inter-
face rather than a class, however specifying a class is supported. The compo-
nent implementation class must implement all the specified service inter-
faces.
```